user_guides/jakartaee/src/main/jbake/content/toc.adoc

```diff
@@ -40,8 +40,8 @@ link:intro.html#GBFOW[1 Introduction]
 ** link:intro.html#GEVCL[1.5 Additional Jakarta EE 10 Platform TCK Requirements]
 ** link:intro.html#GBFQW[1.6 Getting Started With the Jakarta EE 10 Platform TCK Test Suite]
 
-link:rules.html#GBFSN[2 Procedure for Jakarta Platform, Enterprise Edition 10 Certification]
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+link:rules.html#GBFSN[2 Procedure for Jakarta Platform, Enterprise Edition 10.0 Certification]
```

Comment:
From https://github.com/eclipse-ee4j/jakartaee-tck/pull/1042#issuecomment-1139982721 I understood that it is desired to stay as 10...